### ISSUE TITLE
docs

### ISSUE THREAD

**Issue by giovp** [spatial-image/spatial-image#15]
Are there plans to have docs online for spatial-image (or/and multiscale-spatial-image) ? We recently set up docs for [spatialdata](https://scverse-spatialdata.readthedocs.io/en/latest/generated/spatialdata.SpatialData.html) and it'd be great to have intersphinx link to spatial-image.

**Comment by thewtex** [comment]
@giovp yes! We could use ReadTheDocs with the Sphinx configuration as spatialdata.

**Comment by giovp** [comment]
great, before I go ahead with PR, where would you see it best? here or in `multiscale-spatial-image`?
an option would also be to do it in both and then use [rtd subprojects](https://docs.readthedocs.io/en/stable/subprojects.html) such as:
- `spatial-image.readthedocs.io` -> spatial-image classes
- `spatial-image.readthedocs.io/projects/multiscale` -> multiscale docs
- `spatial-image.readthedocs.io/projects/napari` -> napari docs

let me know what you think it's best

**Comment by thewtex** [comment]
@giovp wonderful, thank you!!

I love the RTD subproject layout you proposed.
